build.gradle.kts

```
@@ -2,9 +2,9 @@ import org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
 import org.gradle.api.tasks.testing.logging.TestLogEvent.*
 import java.util.*
 
-project(":commons").version = "2.2.2-SNAPSHOT"
-project(":outbox-kafka-spring").version = "3.2.1-SNAPSHOT"
-project(":outbox-kafka-spring-reactive").version = "3.1.2-SNAPSHOT"
+project(":commons").version = "2.2.2"
+project(":outbox-kafka-spring").version = "3.2.1"
+project(":outbox-kafka-spring-reactive").version = "3.1.2"
```

For existing apps (running spring 6.0 or spring-boot 3.1) the new version is not a drop-in replacement. Therefore I'd propose to increase at least the minor version, and not only the patch version. I'd also be fine with increasing the major version if you'd prefer this.

```suggestion
project(":commons").version = "2.3.0"
project(":outbox-kafka-spring").version = "3.3.0"
project(":outbox-kafka-spring-reactive").version = "3.2.0"
```